File: Backend/src/main/java/com/corp/bookiki/chatbot/entity/FeedbackStatus.java
package com.corp.bookiki.chatbot.entity;

import lombok.Getter;

@Getter
public enum FeedbackStatus {
    PENDING("처리 대기"),
    IN_PROGRESS("처리 중"),
    COMPLETED("처리 완료");

    private final String description;

    FeedbackStatus(String description) {
        this.description = description;
    }
}

File: Frontend/src/types/enums/feedbackStatus.ts
export enum FeedbackStatus {
  PENDING = 'PENDING',
  IN_PROGRESS = 'IN_PROGRESS',
  COMPLETED = 'COMPLETED',
}

export const FeedbackStatusDescriptions: Record<FeedbackStatus, string> = {
  [FeedbackStatus.PENDING]: '처리 대기',
  [FeedbackStatus.IN_PROGRESS]: '처리 중',
  [FeedbackStatus.COMPLETED]: '처리 완료',
};
